Exiled.Permissions/Config.cs

```diff
@@ -27,7 +27,7 @@ public Config()
         }
 
         /// <summary>
-        /// Gets a value indicating whether the debug should be shown or not.
+        /// Gets a value indicating whether if debug should be shown or not.
```

Comment:
I think the previous variant was more correct

Reply:
Reverted